pt/lessons/basics/comprehensions.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
+title: Comprehensions
+category: basics
+order: 13
+lang: pt
+---
+
+Comprehensions de listas são um 'syntactic sugar' (uma forma mais simples de escrever) para realizar loops em Enumerables em Elixir. Nessa lição veremos como podemos fazer iterações e gerar os resultados utilizando comprehensions.
```

Comment:
What do you think about removing `de listas` in those cases?

Or just `Listas de Comprehensions`. ?